{"name": "subtract_300000000_20000000", "status": "failed", "statusDetails": {"message": "AssertionError: assert 100000000 == 280000000", "trace": "self = <testing.test_Calculator.TestCalc object at 0x000001E37D3D3888>\nget_instance = <pythoncode.Calculator.Calculator object at 0x000001E37D3D3D08>\nget_subtract_int_datas_with_fixture = [300000000, 20000000, 100000000]\n\n    @allure.title(\"subtract_{get_subtract_int_datas_with_fixture[0]}_{get_subtract_int_datas_with_fixture[1]}\")\n    @allure.story(\"subtract Integer Function\")\n    def test_subtract_int(self, get_instance, get_subtract_int_datas_with_fixture):\n        subtract_int = get_subtract_int_datas_with_fixture\n>       assert subtract_int[2] == get_instance.subtract(subtract_int[0], subtract_int[1])\nE       assert 100000000 == 280000000\n\ntest_Calculator.py:46: AssertionError"}, "attachments": [{"name": "log", "source": "932f3d35-46b6-4199-b020-edc9b5937760-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_subtract_int_datas_with_fixture", "value": "[300000000, 20000000, 100000000]"}], "start": 1612419968068, "stop": 1612419968068, "uuid": "290ed873-3298-4a49-a03c-71a67f633637", "historyId": "8d7fca0fc46f1f015c3874fc87126fc2", "testCaseId": "c486ae24353ef827fb7e8d41326a9a74", "fullName": "test_Calculator.TestCalc#test_subtract_int", "labels": [{"name": "feature", "value": "计算器"}, {"name": "story", "value": "subtract Integer Function"}, {"name": "suite", "value": "test_Calculator"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "DESKTOP-ALON4FT"}, {"name": "thread", "value": "19128-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_Calculator"}]}